app/routes/events/components/JoinEventForm.tsx

```diff
@@ -172,24 +167,6 @@ const PaymentForm = ({
   </div>
 );
 
-const SpotsLeft = ({ activeCapacity, spotsLeft }: SpotsLeftProps) => {
```

Comment:
In my opinion, we still need this. Any reason for removing?

Reply:
Don't agree, everyone can do the simple math when it says 23/30 places left..

Reply:
It would also make the page more cluttered

Reply:
hmm okay, fair enough hehe